bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/AccordionImpl.java

```diff
@@ -40,26 +48,60 @@
     public final static String RESOURCE_TYPE = "core/wcm/components/accordion/v1/accordion";
 
     @ValueMapValue(optional = true)
-    private String expandedItem;
+    private boolean singleExpansion;
 
     @ValueMapValue(optional = true)
-    private String headingType;
+    private String[] expandedItems;
 
-    private String expandedItemName;
+    @ValueMapValue(optional = true)
+    private String headingElement;
+
+    @ScriptVariable
+    private Style currentStyle;
+
+    private String[] expandedItemNames;
```

Comment:
Having `expandedItemNames` and `expandedItems` is a bit confusing. I would at least add some docs to clarify.